projects/js-packages/components/components/pricing-table/styles.module.scss

```diff
@@ -75,6 +75,8 @@
 			content: '';
 			position: absolute;
 			top: 0;
+			left: var(--padding);
+			right: var(--padding);
```

Comment:
I think this actually breaks the lines on larger screens (pushes them to the right), so maybe we should remove these properties on large viewports?

Reply:
Yeah, good point, I meant to put there. Updated!